src/screen/CreateAccountScreen.tsx

```diff
@@ -39,7 +45,13 @@ export function CreateAccountScreen({navigation}: {navigation: NavigationProp<Ac
   );
 
   const onSubmit = () => {
-    keyring.addUri(mnemonic, account.password, {name: account.title});
+    const {json: key} = keyring.addUri(mnemonic, account.password, {name: account.title});
```

Comment:
I think we need to keep everything in keyring instead of having the keyring accounts duplicated in AccountContext. keyring has a method called `addExternal` that we can use to store the external accounts in it.